server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -565,6 +605,14 @@ public TSDataType getSeriesType(String path) throws MetadataException {
       if (path.equals(SQLConstant.RESERVED_TIME)) {
         return TSDataType.INT64;
       }
+
+      try {
+        MeasurementSchema schema = mRemoteSchemaCache.get(path);
```

Comment:
Then the path simply does not exist.

Reply:
The correct explanation should be added in the catch code block:
```
        // if the mRemoteSchemaCache has no such a path, an IOException will be thrown.
        // we ignore it to get the type from the local mtree.
``` 

By the way, why not check the local MTree before you check the remote?

Reply:
By the way, why not check the local MTree before you check the remote?

Reply:
It is okay, but I think it just repeats the code.
Generally, the chance that the schema is in the cache is much higher than that it is in the MTree because every node only stores a small part of the schemas, and checking the cache is much cheaper than searching the tree. I wonder where your suggestion comes from.

Reply:
I think maybe we need to set the exception type as PathNotExist or PathNotFound in the mRemoteCache.. rather than IOException..